### PR TITLE
The 1st large cleanup of Mobile filters

### PR DESCRIPTION
**Pull request by Yuki2718**
```
! Dead
behacdn.ksmobile.net
cmdts.ksmobile.com
advs2sonline.goforandroid.com
advonline.goforandroid.com
a.6d47c1118927cf53ba9654931df80e8d.com
a.4ee395e5b8a076c7bd8592851ff8cad.com
mobula.ssl2.duapps.com
sync.mobojoy.baidu.com
api.mobojoy.baidu.com
js.mobojoy.baidu.com
a.f590906c3e6c07bb5f80801f2dfc4295.com
a.eed1729eebe45b8cd49e10f2dd80e297.com
pochengweiyuan.com
m.wangliqin.top
m.yangzhenpeng.top
92kyxiz5xg.execute-api.us-west-2.amazonaws.com
0jkro0unul.execute-api.us-west-2.amazonaws.com
nsjpu.com
nbvcb.xyz
cafe-latte.myshoplus.com
eysmaa.pw
eihjefbbbbddeiacedb.ru
zhfosenghtr.com
abtest.mistat.intl.xiaomi.com
opr.formulas.jp
d1zkodo2u3p4pe.cloudfront.net
adbetnet.advertserve.com
aclickoooo.host
883.engine.mobileapptracking.com
ads.cricbuzz.com
adops.cricbuzz.com
nasdfg.com
gocdn.ru
fastmap33.com
m.nbhaosheng168.com
m.jakc0.com
s.nvrentao8.com
xz15801.com
m.mrsasharingspace.com
dc.altamob.com
advantech.vo.llnwd.net
bandai.hs.llnwd.net
embanet.vo.llnwd.net
eyewond.hs.llnwd.net
finedesign.vo.llnwd.net
gulfstream.vo.llnwd.net
gymboree.hs.llnwd.net
harvardbp.vo.llnwd.net
hbpub.vo.llnwd.net
icvdm.vo.llnwd.net
inskin.vo.llnwd.net
msbmopod.vo.llnwd.net
msbrandent.vo.llnwd.net
navigon.vo.llnwd.net
nytelecom.vo.llnwd.net
rbb.ic.llnwd.net
responsys.hs.llnwd.net
surveyor.vo.llnwd.net
toyota.vo.llnwd.net
usarmy.vo.llnwd.net
textron.vo.llnwd.net
xmpie.vo.llnwd.net
sdk.adincube.com
ads.avocarrot.com
moatads.com.edgekey.net
nexage.akadns.net
listat.biz
mobsrv.in
dc.altmob.com
js4386.fujianryt.com
m.12306media.com
sube.puwangkj.com
m.jsqinvest.com
clickkydsp.com
eu-ma.sam4m.com
ad.kixer.com


! in EL
||ad2play.ftv-publicite.fr^
||admob.com^
||smaato.net^

! Redundant
ficbook.net##.rkl-stick-to-bottom
m.hindustantimes.com###stickydivAD

! avclub.com,clickhole.com,deadspin.com,gizmodo.com,jalopnik.com,jezebel.com,kotaku.com,lifehacker.com,splinternews.com,theinventory.com,theonion.com,theroot.com,thetakeout.com##.inline-ad-container
theinventory.com,thetakeout.com,theonion.com,splinternews.com,clickhole.com,avclub.com,deadspin.com,gizmodo.com,jalopnik.com,jezebel.com,kotaku.com,lifehacker.com,theroot.com##.inline-ad-container

! gordonua.com,strana.ua,24tv.ua##amp-iframe
gordonua.com##amp-iframe[src^="https://recreativ.ru/"]

!+ NOT_OPTIMIZED
24tv.ua##.within-ads
lifestyle.24tv.ua##div[class="within-ads"]

! m.lenta.ru##.b-banner-smi2
m.lenta.ru##amp-embed.b-banner-smi2

! gordonua.com,strana.ua,24tv.ua##amp-iframe
amp.strana.ua##amp-iframe[src^="https://mg.lentainform.com/"]

! m.sweclockers.com###topBanner
m.sweclockers.com##.header > .container > #topBanner

! dailywire.com##div[style^="width: 320px; height: 50px;"]
dailywire.com##article[class^="css-"] > div[style="width: 320px; height: 50px; margin: 0px auto 32px;"]

! detik.com##.footer_banner
m.detik.com##.footer_banner

! 9gag.com##.salt-container
m.9gag.com##section.salt-container

```

@ameshkov

### CODE REVIEW

File: MobileFilter/sections/specific_web.txt [hunk 1]
```diff
@@ -1030,13 +1023,11 @@ opencritic.com,www-news18-com.cdn.ampproject.org,eurogamer.net,mumsnet.com,news1
 jianshu.com##.dt-open-bg[data-scene="leftBtn"]
 jianshu.com##.note-comment-above-ad-wrap
 theinventory.com,thetakeout.com,theonion.com,splinternews.com,clickhole.com,avclub.com,deadspin.com,gizmodo.com,jalopnik.com,jezebel.com,kotaku.com,lifehacker.com,theroot.com##main > div > script[type="application/ld+json"] ~ div[class*="-1 "][class*="-0 "]
-theinventory.com,thetakeout.com,theonion.com,splinternews.com,clickhole.com,avclub.com,deadspin.com,gizmodo.com,jalopnik.com,jezebel.com,kotaku.com,lifehacker.com,theroot.com##.inline-ad-container
```
Comment (Alex-302) [inline MobileFilter/sections/specific_web.txt:1033]: image
it is useful

Comment (Alex-302) [reply MobileFilter/sections/specific_web.txt:1033]: If you about two rules - they are grouped. Better leave as is.

Comment (Alex-302) [reply MobileFilter/sections/specific_web.txt:1033]: It seems it is required on theinventory.com only

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
Thanks. 
Restored two rules, which are required.

File: MobileFilter/sections/specific_web.txt (hunk 1 above)
Comment (Yuki2718) [reply MobileFilter/sections/specific_web.txt:1033]: Okay.

**Comment by ameshkov** [comment]
These are alive:
```
wildcard.moatads.com.edgekey.net
d1zkodo2u3p4pe.cloudfront.net
mimanage.behacdn.ksmobile.net
cafe-latte.myshoplus.com
ds-wildcard.moatads.com.edgekey.net
883.engine.mobileapptracking.com
```

**Comment by Yuki2718** [comment]
@ameshkov Restored: https://github.com/AdguardTeam/AdguardFilters/commit/27e230ab74f092b7fdf0fd465a9e58d93f23f792

**Pull request merged**